centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

```diff
@@ -218,21 +218,19 @@
         </group>
       </else-if>
       <else-if type="article-journal" match="any">
-        <group delimiter=", ">
-          <date variable="issued">
-            <date-part name="year"/>
-          </date>
-          <group delimiter=" ">
-            <choose>
-              <if variable="locator" match="any">
-                <text variable="locator" prefix="p. "/>
-              </if>
-              <else-if variable="locator" match="none">
-                <label variable="page" form="short"/>
-                <text variable="page"/>
-              </else-if>
-            </choose>
-          </group>
+        <date variable="issued">
```

Comment:
Please revert this change. This will create erroneous commas after the year if the item has no locator or page.

Reply:
OK